sdk/python/feast/feature_store.py

```diff
@@ -22,13 +22,13 @@ class FeatureStore:
     """
 
     def __init__(
-        self, config_path: Optional[str], config: Optional[Config],
+        self, config_path: Optional[str], config: Optional[RepoConfig],
     ):
         if config_path is None or config is None:
```

Comment:
Wait, isn't this a bug? If we set config_path=None then this exception will raise.

Reply:
I think it is fixed in #1366 ?